pkgs/modules/nodejs/default.nix

```diff
@@ -32,6 +32,8 @@ in
 
   replit = {
 
+    languageServers.typescript-language-server.extensions = [".js" ".jsx" ".ts" ".tsx" ".json"];
```

Comment:
i think it'd be better to set this in the typescript language server module, no?

can probably just set `language = "typescript"` instead of `javascript` there too, but idk what extensions that covers

Reply:
I think this is missing an `(import ../typescript-language-server ...`

Reply:
in config.toml, javascript is `[".js", ".es6", ".mjs", ".cjs"]`, typescript is `[".ts", ".mts", ".cts"]`.

https://github.com/replit/repl-it-web/blob/main/app/filetypes/config.toml

The problem is this is a union of the two. Maybe it should be `languages` , not language.

But the problem of multiple instance of ts-language-server when you have multiple modules that use it still stands. Not sure of a clean solution.

Reply:
> I think this is missing an `(import ../typescript-language-server ...`

That's on line 24.